src/datepicker/Datepicker.vue

```diff
@@ -164,6 +165,19 @@ export default defineComponent({
       required: false,
       default: false,
     },
+    /**
+     * Clears selected date
+     */
+    clearable: {
+      type: Boolean,
+      required: false,
+      default: false
+    }
+  },
+  methods: {
```

Comment:
I would prefer to avoid mixing options API and `setup`. Could you move the method into the setup, like all others?